### PR TITLE
Add Delete Identity examples

### LINKED ISSUE #2261: Add Delete Identity examples
We get asked a fair bit how to delete identities via the API. Lets add these examples to the appropriate page.

To add here: https://docs.flagsmith.com/clients/rest#code-examples

Edge:
DELETE /api/v1/environments/{environment_api_key}/edge-identities/{identity_uuid}/
Core:
DELETE /api/v1/environments/{environment_api_key}/identities/{id}/

Note: This is a copy from issue #336 which has been archived since it was linked to the `flagsmith-docs` repo

### PR DESCRIPTION
**Pull request by novakzaballa**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/deployment/locally-api#pre-commit) to check linting
- [ ] I have filled in the "Changes" section below?
- [ ] I have filled in the "How did you test this code" section below?

## Changes

_Please describe._

## How did you test this code?

_Please describe._

### CODE REVIEW

File: docs/docs/clients/rest.md [hunk 1]
```diff
@@ -305,3 +305,51 @@ for identity in page_of_identities.json()['results']:
...
+
+```python
+import os
+
+import requests
+
+TOKEN = os.environ.get("API_TOKEN")  # obtained from Account section in dashboard
+ENV_KEY = os.environ.get("ENV_KEY")  # obtained from Environment settings in dashboard
+IDENTITY_UUDI = os.environ["IDENTITY_UUDI"] # must (currently) be obtained by inspecting the request to /api/v1/environments/{ENV_KEY}/edge-identities/{IDENTITY_UUDI} in the network console
+BASE_URL = "https://api.flagsmith.com/api/v1"  # update this if self hosting
```
Comment (dabeeeenster) [inline docs/docs/clients/rest.md]: this should be edge api?

File: docs/docs/clients/rest.md [hunk 2]
```diff
@@ -305,3 +305,51 @@ for identity in page_of_identities.json()['results']:
...
+
+```python
+import os
+
+import requests
+
+TOKEN = os.environ.get("API_TOKEN")  # obtained from Account section in dashboard
+ENV_KEY = os.environ.get("ENV_KEY")  # obtained from Environment settings in dashboard
+IDENTITY_ID = os.environ["IDENTITY_ID"] # obtain this from the URL on your dashboard when viewing an identity
+BASE_URL = "https://api.flagsmith.com/api/v1"  # update this if self hosting
```
Comment (dabeeeenster) [inline docs/docs/clients/rest.md:343]: this should be edge api?

File: docs/docs/clients/rest.md [hunk 3]
```diff
@@ -305,3 +305,51 @@ for identity in page_of_identities.json()['results']:
...
+session = requests.Session()
+session.headers.update(
+    {"Authorization": f"Token {TOKEN}", "Content-Type": "application/json"}
+)
+
+# delete the existing identity based on the identity id
+delete_identity_url = f"{BASE_URL}/environments/{ENV_KEY}/identities/{IDENTITY_ID}/"
+delete_identity_response = session.delete(delete_identity_url)
+assert delete_identity_response.status_code == 204
+
```
Comment (dabeeeenster) [inline docs/docs/clients/rest.md]: remove this line

File: docs/docs/clients/rest.md (hunk 1 above)
Comment (novakzaballa) [reply docs/docs/clients/rest.md]: Yes. It is referring to the edge api as you can see in line 327 https://github.com/Flagsmith/flagsmith/blob/cfa760141ae6c896a29f978cd059bf2742924093/docs/docs/clients/rest.md#L327. As far as I understand the base URL is the same for edge and non-edge API calls, right?

File: docs/docs/clients/rest.md (hunk 3 above)
Comment (novakzaballa) [reply docs/docs/clients/rest.md]: Removed

File: docs/docs/clients/rest.md (hunk 2 above)
Comment (novakzaballa) [reply docs/docs/clients/rest.md:343]: Nope, this one is not for edge

File: docs/docs/clients/rest.md (hunk 1 above)
Comment (novakzaballa) [reply docs/docs/clients/rest.md]: Sorry, my bad, I changed the base URL to point edge.api.flagsmith.com

**Review by dabeeeenster — APPROVED** [review]

**Pull request merged**